templates/registration/login.html

```diff
@@ -20,56 +23,89 @@
     href="{% static 'plugins/overlayScrollbars/css/OverlayScrollbars.min.css' %}">
   <!-- Theme style -->
   <link rel="stylesheet" href="{% static 'dist/css/adminlte.min.css' %}">
+
+  <!--Login Page Csss-->
+  <link rel="stylesheet" href="{% static 'dist/css/login-page.css' %}">
+
   <!-- Google Font: Source Sans Pro -->
   <link
     href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700"
     rel="stylesheet">
 </head>
+<body>
 
-<body class="hold-transition login-page">
+<div class="limiter">
+    <div class="container-login100">
+        <div class="wrap-login100">
+            <div class="login100-pic js-tilt">
+                <img src="{% static 'dist/img/login.jpg' %}">
+            </div>
 
-  <div class="login-box">
-    <div class="login-logo">
-      <a href=""><b>{{ school_name }}</b></a>
-    </div>
-    <!-- /.login-logo -->
-    <div class="card">
-      <div class="card-body login-card-body text-center">
-        <p class="login-box-msg">Sign in to start your session</p>
-
-        {% if form.errors %}
-        <p>Your username and password didn't match. Please try again.</p>
-        {% endif %}
-
-        {% if next %}
-        {% if user.is_authenticated %}
-        <p>Your account doesn't have access to this page. To proceed,
-          please login with an account that has access.</p>
-        {% else %}
-        <p>Please login to see this page.</p>
-        {% endif %}
-        {% endif %}
-
-        <form method="post" action="{% url 'login' %}">
-          {% csrf_token %}
-          <table>
-            <tr class="mb-2">
-              <td>{{ form.username.label_tag }}</td>
-              <td>{{ form.username | add_class:"form-control" }}</td>
-            </tr>
-            <tr class="mb-2">
-              <td>{{ form.password.label_tag }}</td>
-              <td>{{ form.password | add_class:"form-control"}}</td>
-            </tr>
-          </table>
-
-          <input type="submit" value="Login" class="btn btn-primary mt-2">
-          <input type="hidden" name="next" value="{{ next }}">
-        </form>
-      </div>
-      <!-- /.login-card-body -->
+            <form class="login100-form validate-form" method="post" action="{% url 'login' %}">
+                {% csrf_token %}
+                <span class="login100-form-title">
+                   {{ school_name }}
+                                </span>
+                <span class="error-msg"> 
+                    {% if form.errors %}
+                    <p class="error-msg">Your username and password didn't match. Please try again.</p>
+                    {% endif %}
+            
+                    {% if next %}
+                    {% if user.is_authenticated %}
+                    <p class="error-msg"> Your account doesn't have access to this page. To proceed,
+                      please login with an account that has access.</p>
```

Comment:
just a correction in upper/lower cases. 

`Please Loging with an Account that has Access!`

Reply:
Okay I'll do that.. is it 'Loging' or 'Login'

Reply:
* Login

Reply:
I have an issue with the image used here. We cannot conclude that this project will always be used for baby schools as the image depicts. Can we replace this with a bootstrap/font-awesome icon and remove the image entirely?

Reply:
Yeah @adigunsherif i can do that. Can you send me the image file or suggest any good icon from fonts icon